src/equals.jl

```diff
@@ -3,6 +3,13 @@
 # equals() now depends on == instead
 # of the other way round.
 function ==(x::Decimal, y::Decimal)
+    # return early on zero
+    zerox = iszero(x)
+    zeroy = iszero(y)
```

Comment:
I keep reading this a `xerox` and `leroy`. I think an underscore would be nice here.

Reply:
Alternatively, call them `xerox` and `leroy`.